docs/tasks/administer-cluster/pvc-protection.md

```diff
@@ -0,0 +1,141 @@
+---
+approvers:
+- msau42
+- jsafrane
+title: Persistent Volume Claim Protection
+---
+
+{% capture overview %}
+{% assign for_k8s_version="v1.9" %}{% include feature-state-alpha.md %}
+
+As of Kubernetes 1.9, persistent volume claims (PVCs) that are actively used by a pod can be protected from pre-mature deletion.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+- A v1.9 or higher Kubernetes must be installed.
+- As PVC Protection is a Kubernetes v1.9 alpha feature it must be turned on:
+1. [Admission controller](/docs/admin/admission-controllers/) must be started with the [PVC Protection plugin](/docs/admin/admission-controllers/#persistent-volume-claim-protection-alpha).
+2. All Kubernetes components must be started with the `PVCProtection` alpha features switched on.
+
+
+{% endcapture %}
+
+{% capture steps %}
+
+## PVC Protection Verification
+
+The below is done in GCE using a Storage Class, however, similar steps can be performed for any type of storage.
```

Comment:
The example below uses a GCE PD StorageClass, however, similar steps can be performed for any volume type.